### PR TITLE
fix(stx): cosmetic fixes to STX

### PR DESCRIPTION
**Pull request by dbrans**
## **Description**
Cosmetic fixes to smart transactions status page.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/27348?quickstart=1)

## **Related issues**

Fixes:
* https://consensyssoftware.atlassian.net/browse/TXL-366
* https://consensyssoftware.atlassian.net/browse/TXL-413
* https://consensyssoftware.atlassian.net/browse/TXL-412

## **Manual testing steps**

1. Go to this page...
2.
3.

### **After**
https://github.com/user-attachments/assets/4cecfe4a-ea8f-47b5-a3d9-d329f18170de

Screenshot with simulation details width fixed:
image

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -5008,18 +5008,11 @@
     "message": "Sudden market changes can cause failures. If the problem continues, reach out to MetaMask customer support."
   },
   "smartTransactionPending": {
```
Comment (dan437) [inline app/_locales/en/messages.json:5010]: When a translation was done into other languages (which is the case with `smartTransactionPending`) and you want to change its value, you have to delete 'smartTransactionPending' from all files and create a new translation key with a different name and new value, only in the `/en` folder.

File: ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx [hunk 2]
```diff
@@ -435,16 +393,13 @@ export const SmartTransactionStatusPage = ({
...
-    timeLeftForPendingStxInSec,
-    isSmartTransactionTakingTooLong,
-    stxDeadline,
-  } = useRemainingTime({
-    isSmartTransactionPending,
-    smartTransaction,
-    stxMaxDeadline,
-    stxEstimatedDeadline,
-  });
+  const { timeLeftForPendingStxInSec, isSmartTransactionTakingTooLong } =
```
Comment (dan437) [inline ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx:396]: Are these 2 still needed?

File: ui/pages/smart-transactions/smart-transaction-status-page/smart-transactions-status-page.test.tsx [hunk 3]
```diff
@@ -209,11 +225,14 @@ describe('SmartTransactionStatusPage', () => {
...
     latestSmartTransaction.status = SmartTransactionStatuses.CANCELLED;
-    requestState.smartTransaction = latestSmartTransaction;
+    requestState.smartTransaction = latestSmartTransaction as SmartTransaction;
     requestState.isDapp = true;
     const store = configureMockStore(middleware)(mockStore);
     const { queryByText, container } = renderWithProvider(
-      <SmartTransactionStatusPage requestState={requestState} />,
+      <SmartTransactionStatusPage
+        requestState={requestState}
+        {...{ onCloseExtension, onViewActivity }}
```
Comment (dan437) [inline ui/pages/smart-transactions/smart-transaction-status-page/smart-transactions-status-page.test.tsx:234]: It's a bit out of scope for this PR, but do you think you could add 2 little additional tests here? One for checking that `onCloseExtension` was called when a user clicked on the `Close extension` button and another one for checking that `onViewActivity` was called after clicking on the `View activity` button.

File: ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx (hunk 2 above)
Comment (dbrans) [reply ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx:396]: Done.

File: ui/pages/smart-transactions/smart-transaction-status-page/smart-transactions-status-page.test.tsx (hunk 3 above)
Comment (dbrans) [reply ui/pages/smart-transactions/smart-transaction-status-page/smart-transactions-status-page.test.tsx:235]: Done.

File: app/_locales/en/messages.json (hunk 1 above)
Comment (dbrans) [reply app/_locales/en/messages.json:5010]: Hey @dan437 – it appears ([slack](https://consensys.slack.com/archives/CTQAGKY5V/p1727805714803069?thread_ts=1704742270.501479&cid=CTQAGKY5V)) we can edit existing english text without deleting the key and creating a new one.

**Review by dan437 — APPROVED** [review]

**Pull request closed without merging**